apps/plugins/kinds/meta.cue

```diff
@@ -5,7 +5,24 @@ metaV0Alpha1: {
 	scope: "Namespaced"
 	schema: {
 		spec: {
```

Comment:
what do you think about the `class` property currently in the `plugin` type? That sounds to me more of a "meta" information? Could we move it here?

Reply:
makes sense to me